javascript-data-model/main.js

```diff
@@ -0,0 +1,104 @@
+/* exported orderHistory */
+
+var orderHistory = [
+  {
+    {
+      orderNumber = 11439416898772232,
+      orderPlaced = '8/4/2020',
+      total = 34.00,
+      shipTo = 'JS Masher'
+    },
+    [
+      {
+        delivered = true,
+        deliveryDate = '8/8/2020',
+        price = 31.55,
+        productImage = 'image.png',
+        productTitle = 'JavaScript for impatient programmers',
+        author = 'Rauschmayer, Dr. Axel',
+        returnWindowClosed = true,
+        returnDate = '9/7/2020',
+        productSupportNeeded = false,
+        writtenProductReview = false
+      }
+    ]
+  },
+  {
+    {
+      orderNumber = 11399842681280257,
+      orderPlaced = '7/19/2020',
+      total = 44.53,
+      shipTo = 'JS Masher'
+    },
+    [
+      {
+        delivered = true,
+        deliveryDate = '7/20/2020',
+        price = 41.33,
+        productImage = 'image.png',
+        productTitle = 'The Timeless Way of Building',
+        author = 'Alexander, Christopher',
+        returnWindowClosed = true,
+        returnDate = '8/19/2020'
+        productSupportNeeded = false,
+        writtenProductReview = false
+      }
+    ]
+  },
+  {
+    {
+      orderNumber = 1142875579059409,
+      orderPlaced = '7/4/2020',
+      total = 17.22,
+      shipTo = 'JS Masher'
+    },
+    [
+      {
+        delivered = true,
+        deliveryDate = '7/7/2020',
+        price = 15.98,
+        productImage = 'image.png',
+        productTitle = 'Gamecube Controller Adapter. Super Smash Bros Switch Gamecube Adapter for WII U, PC. Support Turbo and Vibration Features. No Driver and No Lag-Gamecube Adapter',
+        author = null,
+        returnWindowClosed = true,
+        returnDate = '8/5/2020'
+        productSupportNeeded = false,
+        writtenProductReview = false
+      }
+    ]
+  },
+  {
+    {
+      orderNumber = 11328831772648248,
+      orderPlaced = '7/3/2020',
+      total = 138.93,
+      shipTo = 'JS Masher'
+    },
+    [
```

Comment:
it is good to use an array to separate the items here, but I would attach them to a property on the order object

```
orderNum:
orderPlaced:
items: [
{title:},
{title:}
]

```